locales.json

```diff
@@ -1704,7 +1705,8 @@
       "nearbyPlaces": "Nearby locations",
       "noNearbyPlaces": "No nearby locations found.",
       "alert": "Note",
-      "locationError": "To display nearby locations, location services must be active."
+      "locationError": "To display nearby locations, location services must be active.",
```

Comment:
All keys have to be available at least in German. This is not afaiks

Reply:
You can also omit the key `locationError` here, as the english text as `notAvailable` is also not yet translated. That way it is maybe more consistent in the end.

TL;DR add this only in error

Reply:
This is available in german. This is also not used at all.

Reply:
This line is was not changed by @NKlug, my fault, resolving this